qemu/tests/blockdev_commit_cor.py

```diff
@@ -0,0 +1,49 @@
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitCOR(BlockDevCommitTest):
+
+    def create_snapshots(self, snapshot_tags, device):
+        options = ["node", "overlay"]
+        cmd = "blockdev-snapshot"
+        tag = snapshot_tags[0]
+        params = self.params.object_params(tag)
+        arguments = params.copy_from_keys(options)
+        arguments["overlay"] = self.get_node_name(tag)
+        fmt_node = self.main_vm.devices.get_by_qid(self.device_node)[0]
+        self.base_node = fmt_node.get_param("file")
+        arguments["node"] = self.base_node
+        self.main_vm.monitor.cmd(cmd, dict(arguments))
+        for info in self.disks_info:
+            if device in info:
+                self.generate_tempfile(info[1], tag)
+
+    def commit_snapshots(self):
+        device_tag = self.params["device_tag"].split()[0]
+        device_params = self.params.object_params(device_tag)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        device = self.get_node_name(device_tag)
+        options = ["top-node"]
+        arguments = self.params.copy_from_keys(options)
```

Comment:
looks like top-node is set in line 30, so here arguments = {}

Reply:
Yes, arguments={} here. And we set top-node in line 30.